src/services/lnd/schema.ts

```diff
@@ -0,0 +1,59 @@
+import * as mongoose from "mongoose"
+
+const Schema = mongoose.Schema
+
+const confirmedDetailsSchema = new Schema<LnPaymentConfirmedDetails>({
+  confirmedAt: {
+    type: Date,
+    required: true,
+  },
+  destination: {
+    type: String,
+    required: true,
+  },
+  revealedPreImage: {
+    type: String,
+    required: true,
+  },
+  roundedUpFee: {
+    type: Number,
+    required: true,
+  },
+  milliSatsFee: {
+    type: Number,
+    required: true,
+  },
+  hopPubkeys: [String],
+})
+
+const paymentAttemptSchema = Schema.Types.Mixed
+
+const paymentSchema = new Schema<LnPaymentType>({
+  createdAt: Date,
+  status: {
+    type: String,
+    enum: ["settled", "failed", "pending"],
+    required: true,
+  },
+  paymentHash: {
+    type: String,
+    index: true,
+    unique: true,
+    required: true,
+  },
+  paymentRequest: {
+    type: String,
+    unique: true,
```

Comment:
do we need an index on this, if we already have an index in the hash?

the index on the hash will it so that it's not possible to add the same paymentRequest twice